test/Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM node:14.21.3-bullseye
 
 RUN apt-get update && \
     apt-get --no-install-recommends -y install \
-        docker.io=20.10.5+dfsg1-1+deb11u2 \
-        chromium=116.0.5845.110-1~deb11u1 \
-        python3-pip=20.3.4-4+deb11u1 && \
+        docker.io \
+        chromium \
+        python3-pip && \
     ln -sf /usr/bin/python3 /usr/bin/python && \
     rm -rf /var/lib/apt/lists/* && \
     pip3 install --no-cache-dir --upgrade \
-        pip==23.2.1 \
-        setuptools==68.1.2
+        pip \
+        setuptools
```

Comment:
Do we not need to pin versions after all, then?

Reply:
Nope, and you were right, we probably shouldn't! 

There were a series of small issues here starting with the path to the `.hadolint` config file being wrong, hence causing linting warnings which otherwise the `.hadolint` config had turned off that led to me following and pinning the versions 🤦🏼‍♂️ It's all sorted now, and hopefully simpler to read. Gonna request a re-review, if you don't mind.